web/web/src/lib/utils/initial.js

```diff
@@ -570,10 +566,64 @@ const relationalColumnTypeMap = {
   ]
 }
 
-export const getRelationalColumnTypeMap = catalog => {
-  if (Object.keys(relationalColumnTypeMap).includes(catalog)) {
+export const getRelationalColumnType = catalog => {
+  if (Object.hasOwn(relationalColumnTypeMap, catalog)) {
     return relationalColumnTypeMap[catalog]
   }
 
   return []
 }
+
+const relationalTablePropInfoMap = {
+  'lakehouse-iceberg': {
+    reserved: ['location'],
```

Comment:
Aren't there some properties missing about iceberg: https://gravitino.apache.org/docs/0.7.0-incubating/lakehouse-iceberg-catalog#table-properties

Reply:
Sorry, I missed that. Will update it.

Some properties, like `serde-name`, are not in the documentation, but are they also immutable?